helm/trust-registry/conf/dev/values.yaml

```
@@ -22,6 +22,10 @@ ingress:
     hosts:
       - host: ssi-trust-registry.cloudapi.dev.didxtech.com
 
+config:
+  frontendUrl: https://ssi-trust-registry.cloudapi.dev.didxtech.com
+  nextPublicBackendUrl: https://ssi-trust-registry.cloudapi.dev.didxtech.com/api
```

We must set just `https://ssi-trust-registry.cloudapi.dev.didxtech.com` because the frontend app already adds the `/api` suffix.
```suggestion
  nextPublicBackendUrl: https://ssi-trust-registry.cloudapi.dev.didxtech.com
```